doc/specifications/rest/REST-API-V2.rst

```diff
@@ -882,6 +887,91 @@ XML Example
+         :application/vnd.ez.api.ContentInfo+xml:  if set all informations for the content object (excluding the current version) are returned in xml format (see Content_)
+         :application/vnd.ez.api.ContentInfo+json:  if set all informations for the content object (excluding the current version) are returned in json format (see Content_)
+    :If-None-Match: <etag> If the provided etag matches the current etag then a 304 Not Modified is returned. The etag changes if the meta data was changed - this happens also if there is a new published version.
+:Parameters:
+    :languages: (comma separated list) restricts the output of translatable fields to the given languages
+:Response:
+
+
+.. code:: http
+
+          HTTP/1.1 200 OK
+          ETag: "<ETag>"
+          Accept-Patch: application/vnd.ez.api.ContentUpdate+(json|xml)
+          Content-Type: <depending on accept header>
+          Content-Length: <length>
+.. parsed-literal::
+Content_
+
+:Error Codes:
+    :401: If the user is not authorized to read  this object. This could also happen if there is no published version yet and another user owns a draft of this content
+    :404: If the ID or the user hash is not found
+
+XML Example
+'''''''''''
+
+.. code:: http
+
+    GET /content/objects/23/permissions/38015b703d82206ebc01d17a39c727e5 HTTP/1.1
+    Accept: application/vnd.ez.api.ContentInfo+xml
+    If-None-Match: "12340577"
+
+.. code:: http
+
+    HTTP/1.1 200 OK
+    ETag: "12345678"
+    Accept-Patch: application/vnd.ez.api.ContentUpdate+xml;charset=utf8
+    Content-Type: application/vnd.ez.api.ContentInfo+xml
+    Content-Length: xxx
+
+.. code:: xml
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <Content href="/content/objects/23" id="23"
+      media-type="application/vnd.ez.api.Content+xml" remoteId="qwert123">
+      <ContentType href="/content/types/10" media-type="application/vnd.ez.api.ContentType+xml" />
+      <Name>This is a title</Name>
+      <Versions href="/content/objects/23/versions" media-type="application/vnd.ez.api.VersionList+xml" />
+      <CurrentVersion href="/content/objects/23/currentversion"
+        media-type="application/vnd.ez.api.Version+xml"/>
+      <Section href="/content/sections/4" media-type="application/vnd.ez.api.Section+xml" />
+      <MainLocation href="/content/locations/1/4/65" media-type="application/vnd.ez.api.Location+xml" />
+      <Locations href="/content/objects/23/locations" media-type="application/vnd.ez.api.LocationList+xml" />
+      <Owner href="/user/users/14" media-type="application/vnd.ez.api.User+xml" />
+      <lastModificationDate>2012-02-12T12:30:00</lastModificationDate>
+      <publishedDate>2012-02-12T15:30:00</publishedDate>
+      <mainLanguageCode>eng-US</mainLanguageCode>
+      <currentVersionNo>1</currentVersionNo>
+      <alwaysAvailable>true</alwaysAvailable>
+      <ContentPermissions>
+        <canRead>true</canRead>
```

Comment:
Do we want something more generic like `<function name="canRead">true</function>`? I thought the current way makes the schema simpler, since some "functions" can have specific details, others are just booleans.